src/Pendulum/exploratory/sketch.js

```diff
@@ -168,34 +265,84 @@ function createWorld() {
      Bodies.rectangle(800, 300, 50, 600, { isStatic: true }),
      Bodies.rectangle(0, 300, 50, 600, { isStatic: true })
   ]);
-
-  var massVal = parseInt(massSlider.noUiSlider.get(), 10) / 1000;
-  var angleVal = parseInt(angleSlider.noUiSlider.get(), 10);
-  var lengthVal = parseFloat(lengthSlider.noUiSlider.get(), 10);
-
   var xCoordProtractor = 400;
   var yCoordProtractor = 50;
 
-  var xCoordBody = 400 - ((lengthVal * PTM) * Math.sin(angleVal * Math.PI / 180));
-  var yCoordBody = (lengthVal * PTM) * Math.cos(angleVal * Math.PI / 180) + yCoordProtractor;
+  var lengthVal = parseFloat(lengthSlider.noUiSlider.get(), 10);
+
+  // first pendulum
+  var massVal = parseInt(massSlider.noUiSlider.get(), 10) / 1000; // convert to kg
+  var angleVal = parseInt(angleSlider.noUiSlider.get(), 10);
+  var xCoordBody = calcXCoord(lengthVal, angleVal);
+  var yCoordBody = calcYCoord(lengthVal, angleVal, yCoordProtractor);
 
   pendulum.pendulumBody = Bodies.circle(xCoordBody, yCoordBody, 30, {
     mass: massVal,
     frictionAir: 0,
-    interia: Infinity });
+    interia: Infinity,
+    render: {
+      fillStyle: "rgb(97, 181, 255)"
+    }});
 
-  // set the angle
   let protractor = Bodies.circle(xCoordProtractor, yCoordProtractor, 20, { isStatic: true});
 
-  World.add(engine.world, [pendulum.pendulumBody, protractor]);
-
   pendulum.pendulumString = World.add(engine.world, Constraint.create({
     bodyA: protractor,
     bodyB: pendulum.pendulumBody,
     length: 0
   }));
 
   pendulum.pendulumStringLength = pendulum.calculateStringLength(protractor.position, pendulum.pendulumBody.position);
+
+  // second pendulum
+  var massVal2 = parseInt(mass2Slider.noUiSlider.get(), 10) / 1000; // convert to kg
+  var angleVal2 = parseInt(angle2Slider.noUiSlider.get(), 10);
+  var xCoordBody2 = calcXCoord(lengthVal, angleVal2);
+  var yCoordBody2 = calcYCoord(lengthVal, angleVal2, yCoordProtractor);
+  var restVal = parseFloat(corSlider.noUiSlider.get());
+
+  pendulum2.pendulumBody = Bodies.circle(xCoordBody2, yCoordBody2, 30, {
+     mass: massVal2, 
+     frictionAir: 0, 
+     interia: Infinity,
+     friction: 0,
+     restitution: restVal,   // matter should take the max rest val of 2 objects
+     render: {
+      fillStyle: "rgb(64, 173, 111)"
+    }
+    });
+
+  // add first pendulum and protractor
+  World.add(engine.world, [pendulum.pendulumBody, protractor]);
+
+  // add the second pendulum if selected in the dropdown
+  if(numWeightsDropdown.value == "2") {
+    World.add(engine.world, [pendulum2.pendulumBody]);
+    pendulum2.pendulumString = World.add(engine.world, Constraint.create({
+      bodyA: protractor,
+      bodyB: pendulum2.pendulumBody,
+      length: 0,
+    }));
+  }
+  
+}
+
+/**
+ * add a second pendulum to the world 
```

Comment:
Might not need these function stubs but I left them here incase we didn't wanna cram it all in createWorld